### PR TITLE
SSH Command expects optional parameter and use default instead

### PR DESCRIPTION
**Pull request by stevenklar**
| Q             | A
| ------------- | ---
| Bug fix?      | No
| New feature?  | Yes
| BC breaks?    | No
| Deprecations? | No
| Fixed tickets | N/A

Make SshCommand host parameter optional when only using the default (first) host.

> dep ssh

### CODE REVIEW

File: src/Console/SshCommand.php [hunk 1]
```diff
@@ -49,7 +49,10 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $host = $this->deployer->hosts->get($input->getArgument('hostname'));
+        $firstHostname = $this->deployer->hosts->getIterator()->current()->getHostname();
```
Comment (staabm) [inline src/Console/SshCommand.php]: I guess this only makes sense when there is actually only 1 host, right?

Comment (stevenklar) [inline src/Console/SshCommand.php]: Yes. That's correct.
If multiple hosts are given the first one will be choosen.

Comment (staabm) [inline src/Console/SshCommand.php]: what I meant: this fallback to the first hostname only makes sense where there is just one defined.

using the first when several are defined doesnt sound like a usefull guess.

Comment (stevenklar) [inline src/Console/SshCommand.php]: What does sound like a good guess for you?
First thing which comes in mind for me would be an addtional configuration.

default_stage for example.

Comment (staabm) [inline src/Console/SshCommand.php]: atm I can't think of a good guess. using the only one defined sounds straight forward.

in  a interactive mode you might "ask the user" to choose from the defined list, but non-interactive I dont think you can pre-dict which host is the "correct" one.

Comment (stevenklar) [inline src/Console/SshCommand.php]: Would you see that interactive mode necessary or could that be an enhancement
`dep ssh -i`

Comment (staabm) [inline src/Console/SshCommand.php]: lets see what @antonmedv thinks about such a feature

Comment (stevenklar) [inline src/Console/SshCommand.php]: Allright. Maybe short explaination why I came up with this simple feature.

When using `dep ssh` I expected initial that it directly connects me instead of writing `dep ssh user@my-little-pony.de`all the time.

Comment (antonmedv) [inline src/Console/SshCommand.php]: I think it's ok only if there only one host. Otherwise it's implicitly, and explicitly is always better. Inveractive mode maybe solution, but i don't like -i option, maybe it's better to be by default:
* dep ssh - interactive
* dep ssh host

Comment (stevenklar) [inline src/Console/SshCommand.php]: I don't quite understand the `dep ssh - interactive`path.
User is typing `dep ssh - interactive`and get a list of hosts?

That would be an enhancement for another pull request. Would it fit with an additional "check if there is only one host registered"?

Comment (antonmedv) [inline src/Console/SshCommand.php]: `dep ssh` - leads to interactive choose

**Comment by antonmedv** [comment]
Done 7f74425d886aceddba3381597762c072316a0330
